### DISCUSSION TITLE
How to obtain or save the target optimization value during the iteration process

### DISCUSSION THREAD

**Discussion by styw1234** [yalmip/YALMIP#1412] [General discussion]
How to obtain or save the target optimization value during the iteration process in yalmip？

Is there any command or variable can get the objective value before the final optimization results.

**Comment by johanlofberg** [comment]
Unclear what you actually want to do, or which iteration or solver you are talking about

**Comment by styw1234** [comment]
In yalmip, we can set 'savesolveroutput' as 1 to get the final output of the solver after the iteration.

Then how to get the output of the solver during the iteration? That is to say, how to get the output of the solver after each iteration and save the outputs to a variable?

**Reply by johanlofberg** [reply]
https://yalmip.github.io/Extracting

_If the information of interest isn’t available here, it is not possible to extract it through the MATLAB interface._

**Reply by styw1234** [reply]
OK！
